pulser/simulation.py

```diff
@@ -155,8 +177,8 @@ def make_vdw_term():
             min_dist = 2 * self._seq._device.max_radial_distance
             for q1, q2 in itertools.combinations(self._qdict.keys(), r=2):
                 dist = np.linalg.norm(
-                        self._qdict[q1] - self._qdict[q2])
-                U = 0.5 * 5.008e6 / dist**6  # = U/hbar
+                    self._qdict[q1] - self._qdict[q2])
+                U = 0.5 * self._seq._device.interaction_coeff / dist**6
```

Comment:
@sebgrijalva I am the one who did that change. Seems ok for you to use device.interaction_coeff ?

Reply:
Yes, thanks, I agree it should be taken from the device.